content/news/2021-new-marketing-members/_index.md

```diff
@@ -0,0 +1,38 @@
+
+
+![](otavio.png)
+
+
+Otavio says, “I’m pleased to be part of the Jakarta EE Marketing and Branding Committee and help in shaping the strategy for our whole community to promote, share, engage with events about technology and bring even more visibility to the progress we’re making. This community is unique and will be a powerful force for the future of technology development because of the collaborative approach that drives us - where everybody can cooperate, help, and share their knowledge to create a community of success around Jakarta EE”
+
+Otavio will also contribute Platform.sh's know-how, which has helped thousands of companies in the cloud adoption globally, the goal is to bring even more maturity and credibility with [guides](https://docs.platform.sh/guides.html), [templates](https://docs.platform.sh/development/templates.html#java), and cases, indeed, in the Java/Jakarta EE side as well. Furthermore, Platform.sh has a [community sponsorship program](https://docs.google.com/document/d/1yXrjk4obCOaRThqUiL0h3llOldOHbGVXAyYYHEFjcko/edit#heading=h.fgys9p2kbv6) to help the community and the open-source.
+
+Tetiana Fydorenchyk is Vice President of Marketing at [Jelastic](https://jelastic.com/), a multi-cloud PaaS provider serving developers and hosting companies around the world. The company twice got [Duke’s Choice Award](https://jelastic.com/blog/duke-choice-award-2018-winner/) for its contribution to Java enhancement and cloud adoption. Tetiana provides executive leadership and management of Jelastic inbound and out-bound marketing activities worldwide, including: corporate, partner/channel, content and product marketing. She produces [articles](https://jelastic.com/blog/) on tech and business topics related to cloud computing, containers, DevOps, microservices, multi-cloud and others. Also, currently Tetiana is a member of Jakarta EE and Microprofile Working Groups.
+
+Tetiana collaborates with leaders in the Java community and cloud customers on a daily basis. She develops blogs, arranges conferences, webinars, user groups meetups and other events designed to support the Java community in its growth and innovation. She has expertise in content creation, online events, public relations and social media, and looks forward to using these skills on behalf of the Jakarta EE project.
+
+Tetiana says, “I’m thrilled to be part of the Jakarta EE Marketing Committee and leverage my network and marketing communications to help this exciting initiative. The main purpose is to accelerate and sustain the growth of the Jakarta EE community by making a valuable practical contribution such as spreading the word, creating content, participating in the events and collaborating on the future plans.”
+
+
+<img src="tetiana.png" width="720px">
```

Comment:
We currently do not allow html in markdown files:
https://github.com/EclipseFdn/hugo-solstice-theme#known-issues

However, I am starting to think that this could make things easier for contributors. 

Can someone create a separate issue and ask that we enable this on jakarta.ee if folks are interested in this functionnality?

Reply:
will replace for now, with md and limit size on css side

Reply:
done